NetworkStack/NetworkRequests.swift

```diff
@@ -24,21 +35,33 @@ public struct NetworkAPIRequests: NetworkRequests {
     
     var network = Network()
     
-    fileprivate var defaultSession: URLSession? {
+    fileprivate func defaultSession() throws -> URLSession? {
         guard let appNetworkState = AppNetworkState.currentAppState else { return nil }
-        guard let accessToken = appNetworkState.accessToken else { return nil }
-        let configuration = URLSessionConfiguration.default
-        configuration.httpAdditionalHeaders = ["content-type": "application/json", "Authorization": "Bearer \(accessToken)"]
+        guard let accessToken = try appNetworkState.accessToken() else { return nil }
+        let configuration = URLSessionConfiguration.ephemeral
+        
+        let headers: [String:String] = [
+            "Accept": "application/json",
+            "Authorization": "Bearer \(accessToken)",
+            "Accept-Language": Bundle.main.acceptLanguages,
+            "X-Client-Id": appNetworkState.appVersionSlug,
+            "X-Hydrate": "true"
+        ]
+        configuration.httpAdditionalHeaders = headers
+        
         configuration.timeoutIntervalForRequest = 10.0
         return URLSession(configuration: configuration)
     }
-
-    fileprivate var activeSession: URLSession? {
-        return overrideSession ?? defaultSession
+    
+    fileprivate func activeSession() throws -> URLSession? {
+        if overrideSession != nil {
```

Comment:
Can we not do the following?

```suggestion
        return overrideSession ?? try defaultSession()
```